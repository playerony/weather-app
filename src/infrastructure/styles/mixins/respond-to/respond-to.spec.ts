import { toMatchSnapshot, functionImportTest } from '@utils';
import { respondToMin, respondToMax } from './respond-to.mixin';

describe('respondToMin Mixin', () => {
  functionImportTest(respondToMin.large);
  toMatchSnapshot(() => respondToMin.large`background-color: red;`);
});

describe('respondToMax Mixin', () => {
  functionImportTest(respondToMax.large);
  toMatchSnapshot(() => respondToMax.large`background-color: red;`);
});
