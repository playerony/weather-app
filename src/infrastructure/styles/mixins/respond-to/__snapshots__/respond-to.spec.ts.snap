// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`respondToMax Mixin snapshot test should match snapshot 1`] = `
Array [
  "
      @media (",
  "max-width",
  ": ",
  "1919",
  "px) {
        ",
  "background-color: red;",
  ";
      }
    ",
]
`;

exports[`respondToMin Mixin snapshot test should match snapshot 1`] = `
Array [
  "
      @media (",
  "min-width",
  ": ",
  "1919",
  "px) {
        ",
  "background-color: red;",
  ";
      }
    ",
]
`;
